deploy-util.sh

```diff
@@ -31,6 +32,22 @@ function show_help() {
     done
 }
 
+function snitch_deploy() {
+    set -e
+    curl -sSL https://github.com/lucasgomide/snitch/releases/download/0.4.0/snitch_0.4.0_linux_amd64.tar.gz | tar xz
+    source ./APP_EXTRA_ENV
+    cat > snitch.yml << EOF
+hangouts_chat:
```

Comment:
What about the Sentry?

Reply:
Sentry is not notified: it get the app-version on each request to store an event.
`deploy-utils` only sets this value as environment var, sentry app specific setup should handle that.